### PR TITLE
add search functionality to OMEdit documentation

### PR DESCRIPTION
**Pull request by arun3688**

### CODE REVIEW

**Comment by adeas31** [comment]
Well I am not sure if we should add the `advanced search` documentation to users guide. It is only for MetaModelica or text files. Maybe we should add a section about MetaModelica support in the users guide and then add the search section in it.

**Comment by arun3688** [comment]
@adeas31 So where should we add MetaModelica support in users guide, in OMEdit section or seperate heading, because peter asked to include search documentation in OMEdit.

**Comment by adeas31** [comment]
Maybe we should add a section called `Using OMEdit as Text Editor` after `State Machines` and then add `Advanced Search` as a sub section inside it but you have to first also write a bit about how to load/add a directory/file.

**Comment by arun3688** [comment]
@adeas31 In the OMEdit documentation i see there is already some contents written about the Text Editor and other editor, I Think we can make Text -Editor as top level section and include search-features under that heading What is your opinion ?

**Comment by adeas31** [comment]
No, the contents that you are referring are part of OMEdit settings and they should be there.

**Comment by arun3688** [comment]
@adeaI I have added some texts about Using OMEdit as text Editor can you look into it

File: UsersGuide/source/omedit.rst [hunk 1]
```diff
@@ -1534,3 +1534,66 @@ setting that flag in the *Tools > Options > Simulation* dialog.
   :name: omedit-state-machine-simulation-settings
 
   Ensure (at least) Modelica v3.3 support.
+  
+Using OMEdit as Text Editor
+---------------------------
+OMEdit can be be used as a Text editor. Currently support for editing MetaModelica,Modelica and C/C++ 
+are available with syntax highlighting and autocompletion of keywods and types. Additionaly the Modelica
```
Comment (adeas31) [inline UsersGuide/source/omedit.rst]: `keywords` instead of `keywods`

**Comment by arun3688** [comment]
@adeas31 fixed the typos, I don't have access to merge the pull request, can you push it

**Pull request merged**